src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
 
 <img
-  title="NRAI_Decisions_Index.png"
-  alt="A screenshot that shows applied intelligence correlating incident data in the decisions UI."
-  src={nraiDecisionsIndex}
+  title="NRAI_Decisions_Page.png"
+  alt="A screenshot that shows applied intelligence decisions UI."
+  src={nraiDecisionsPage}
 />
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Applied intelligence > Incident intelligence > Decisions**: Our UI shows how each decision correlates incidents.
 </figcaption>
 
-## Key concepts [#key-concepts]
+### What is correlation and how does it work? [#what-is-correlaton]
 
-Here are some key concepts for understanding our decisions logic:
+Your most recent and active incidents are available for our correlation logic. For example, let's say your system has received two alerts saying a synthetic monitor is failing in Australia and London. These two alerts will have created their own, unique issues. Those issues will generate their own, unique incidents based on your teams existing [incident creation policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/#preference-target). New Relic's correlation logic will then test those incidents against each other to find similarities. In this case, it's the same monitor that is failing across multiple locations, so New Relic will merge both incidents into a single incident that contains each relevant event.
 
-### What is "correlation" and how does it work?
+When we correlate events, we check every pair of combinations against each other and combine as many as possible. For example:
+* Our algorithm correlates incident A and B (call it "AB").
+* Our algorithm correlates incident B and C (call it "BC").
+* Because B is present in both issues, the algorithm then correlates all three incidents together into one issue.
 
-For the [incident events](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed) being sent into incident intelligence from your various alerting engines, the most recently created and active incidents are available for correlation. Correlation occurs between two events whenever the criteria in any decision are met.
+## Configure correlation policy [#configure-correlation]
+
+To enable correlation on [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts) based issues, you'll need to connect to correlation for the respective [alert policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#alert-policy-name).
```

Comment:
should that be 'alert-based', with hyphen?

Reply:
yes, you're right.